wandb/apis/public.py

```diff
@@ -5192,3 +5226,157 @@ def call(
             cuda=cuda,
         )
         return queued_run
+
+
+class ArtifactPortfolio(Attrs):
```

Comment:
How might we restructure this to share common features between Portfolios and Sequences? Notice in `class ArtifactCollection` (which uses Sequence GQL endpoints), we have a methods like `load`, `version`, etc... As a user, they should not need to think about if something is a sequence or a portfolio. Regardless, for example, they should be able to get the versions. 

This relates to the bigger picture we have been talking about - anytime you find yourself writing something Portfolio or Sequence specific, it is probably too specific. the only differences between the two are:
1. A single field in the DB
2. An Artifact version always belongs to 1 and only 1 sequence, but may belong to 0-to-many portfolios. 

The second rule is just a business rule on memberships. Every other class/method/api we build should be the same between the two.

Reply:
Moving this to be handled by the LinkArtifact resolver on the backend instead. Thanks for looking into this Tim!